MC/CustomGenerators/PWGHF/Pythia_pp_XiC.C

```diff
@@ -0,0 +1,24 @@
+AliGenerator *GeneratorCustom(TString opt =)
```

Comment:
I think, one needs to add a default argument here: String opt = ""

Reply:
Hello, sorry I missed this message. The correction is done.